driver-core/src/test/unit/com/mongodb/internal/connection/InternalStreamConnectionInitializerSpecification.groovy

```diff
@@ -243,6 +256,94 @@ class InternalStreamConnectionInitializerSpecification extends Specification {
                                  [true, false]].combinations()
     }
 
+    @IgnoreIf({ !serverVersionAtLeast(4, 3) })
```

Comment:
If this is a unit test (and therefore doesn't talk to a server) there's no need for these annotations